tests/phpunit/integration/Core/Util/Migration_1_121_0Test.php

```diff
@@ -0,0 +1,190 @@
+<?php
+/**
+ * \Google\Site_Kit\Tests\Core\Util\Migration_1_119_0Test
+ *
+ * @package   Google\Site_Kit\Tests\Core\Util
+ * @copyright 2024 Google LLC
+ * @license   https://www.apache.org/licenses/LICENSE-2.0 Apache License 2.0
+ * @link      https://sitekit.withgoogle.com
+ */
+
+namespace Google\Site_Kit\Tests\Core\Util;
+
+use Google\Site_Kit\Context;
+use Google\Site_Kit\Core\Authentication\Authentication;
+use Google\Site_Kit\Core\Storage\Options;
+use Google\Site_Kit\Core\Storage\User_Options;
+use Google\Site_Kit\Core\Util\Migration_1_121_0;
+use Google\Site_Kit\Modules\Analytics_4;
+use Google\Site_Kit\Tests\Fake_Site_Connection_Trait;
+use Google\Site_Kit\Tests\TestCase;
+
+class Migration_1_121_0Test extends TestCase {
+
+	use Fake_Site_Connection_Trait;
+
+	/**
+	 * @var integer
+	 */
+	protected $user_id;
+
+	/**
+	 * @var Context
+	 */
+	protected $context;
+
+	/**
+	 * @var Options
+	 */
+	protected $options;
+
+	/**
+	 * @var User_Options
+	 */
+	protected $user_options;
+
+	/**
+	 * @var Authentication
+	 */
+	protected $authentication;
+
+	/**
+	 * @var Analytics_4
+	 */
+	protected $analytics_4;
```

Comment:
We should be able to get away with using the `Modules\Analytics_4\Settings` class only. What do you think?

```suggestion
	/**
	 * @var Settings
	 */
	protected $analytics_settings;
```

Reply:
Yes, we actually can we a bit of the tweak for exposing the default values